addins/Cake.Endpoint.yml

```diff
@@ -2,7 +2,7 @@ Name: Cake.Endpoint
 NuGet: Cake.Endpoint
 Assemblies:
 - "/**/Cake.Endpoint.dll"
-Repository: https://github.com/cake-contrib/Cake.Endpoint/
+Repository: https://github.com/cake-contrib/Cake.Endpoint
```

Comment:
@Jericho wondering if this is diff "enough" to warrant an update?

Reply:
We just need to decide what we want in our YAML files and I can make sure to standardize URLs. What do you think our standard should be: with or without slash?